src/style/abstracts/_mixins.scss

```
@@ -0,0 +1,43 @@
+@mixin mobile-sm {
+  @media (min-width: 375px) {
+    @content;
+  }
+}
+@mixin mobile-md {
+  @media (min-width: 411px) {
+    @content;
+  }
+}
+@mixin tablet-sm {
+  @media (min-width: 768px) {
+    @content;
+  }
+}
+@mixin tablet-md {
+  @media (min-width: 910px) {
+    @content;
+  }
+}
+@mixin laptop {
+  @media (min-width: 1366px) {
+    @content;
+  }
+}
+@mixin desktop {
+  @media (min-width: 1600px) {
+    @content;
+  }
+}
+
+// modal
+
+@mixin select_label {
+  position: absolute;
+  top: -21%;
+  left: 2%;
+  color: var(--gray-60);
+  font-size: 12px;
+  font-weight: 500;
+  padding: 2px 6px;
+  background-color: var(--white-100);
+}
```

This is not used currently
```suggestion
// modal

@mixin select_label {
  position: absolute;
  top: -21%;
  left: 2%;
  color: var(--gray-60);
  font-size: 12px;
  font-weight: 500;
  padding: 2px 6px;
  background-color: var(--white-100);
}
```